.github/workflows/ci-test.yml

```diff
@@ -81,7 +81,7 @@ jobs:
 
   non-rust-lint:
     runs-on: ubuntu-latest
```

Comment:
@matbd  you'll likely want this to be ubuntu-latest-xl to get a much larger machine than you would get with this.

Reply:
Oops I read too fast. Thanks